packages/blocks/src/root-block/root-service.ts

```diff
@@ -140,6 +135,8 @@ export abstract class RootService extends BlockService<RootBlockModel> {
   }
 
   override mounted() {
+    DocModeService.add(this.std);
+
```

Comment:
May be we need to refactor the initialize logic of `std` first. I want to remove the dependence on component mounted.